libdrgn/program.c

```diff
@@ -487,9 +487,10 @@ static struct drgn_error *find_elf_symbol(Elf *elf, Elf_Scn *symtab_scn,
 }
 
 static const char * const vmlinux_paths[] = {
-	"/usr/lib/debug/lib/modules/%s/vmlinux",
```

Comment:
I think my reasoning with the order was that the files under /usr/lib/debug always have debug information, but maybe in /boot or /lib/modules they don't, so it'd be best to check /usr/lib/debug first. Could you keep the order and maybe add a comment like "Check /usr/lib/debug first, since those will always have debug information."